onnxruntime/python/tools/transformers/huggingface_models.py

```diff
@@ -158,4 +158,7 @@
     ),
     # "google/pegasus-xsum": (["input_ids"], 11, False, "bert"),
     # "google/pegasus-large": (["input_ids"], 11, False, "bert"),
+
+    # ViT
+    "google/vit-base-patch16-224": (["input_ids"], 12, False, "bert"),
```

Comment:
Better to add a new model type for optimizer instead of using "bert" here. You can derive OnnxModelVit from OnnxModelBert.